src/algebra/category/Group/images.lean

```diff
@@ -85,7 +85,21 @@ noncomputable instance : has_image f :=
   { lift := image.lift,
     lift_fac' := image.lift_fac } }
 
-noncomputable instance : has_images.{0} AddCommGroup.{0} :=
+noncomputable instance : has_images AddCommGroup.{0} :=
 { has_image := infer_instance }
 
+-- We'll later get this as a consequence of `[abelian AddCommGroup]`,
+-- but I'd like it now, in order to be able to demonstrate cohomology calculations.
```

Comment:
What do you mean by "now"?  Saturday? `images.lean`?

I'm okay with merging this today to fill the gap until we have the general instance.

My instinct would be to avoid duplicate instances.  As you've written, they are likely not definitionally equal to the general one.

Reply:
Sorry, shouldn't have written "now". Let me improve the comment.